docs/review/README.md

```diff
@@ -2,184 +2,158 @@
 
 ![Preview background example](../img/getting-started/preview_background_example.png)
 
 See the **Customization** section for more details.
 [Pipeline Customization](../customization-pipeline/README.md)
+<!-- Dead link, need to confirm where this goes to -->
 
-You can draw directly on the preview with the **Pencil** tool and select a color ![Pencil](../img/getting-started/draw.png).
+You can select a color to use and then draw directly on the preview with the **Pencil** tool ![Pencil](../img/getting-started/draw.png).
 
 ![review drawing](../img/getting-started/review_comment.png)
 
-You can also add **text** on the frame.
+You can also annotate the frame with **text**.
 
 ![review drawing text](../img/getting-started/review_comment_text.png)
 
-
-If you need to see the preview without your comment, **hide your annotation**, you can click on the **Fountain Pen** button.
+If you'd like to temporarily **hide annotations**, click the **Fountain Pen** button.
 ![Hide Annotation](../img/getting-started/hide_annotation.png)
 
+If you would like to refer back to a specific frame in your comments, you can easily tag a frame by typing `@`, which opens a sub-menu with the team list, and then start typing **frame**.
 
-You can tag a frame (whatever version you are in), type the `@`and it opens a sub-menu with the team list, and start typing **frame**.
+This will then add a timestamp of the current frame to this comment, and clicking on that timestamp will jump to said frame.
 
 ![Tag a frame](../img/getting-started/tag_frame.png)
 
-
 You can also export your drawings on each frame by attaching the **screenshots as attachments**.
 
 Click on the **Add attachment** button.
 
 ![attachment](../img/getting-started/attachment_retake.png)
 
-You can also choose to **Attach snapshots from your annotation** on the attachment option.
+You can also choose to **Attach snapshots from your annotation** in the attachment options.
+
+<!-- This doesn't make sense, need to confirm what this is referring to -->
 
 ![Attachment snapshot](../img/getting-started/screenshot_attachment.png)
 
-Once you click the button, Kitsu will grab all the frames with annotation and display the result. You can publish them with **Confirm**.
+Once you click the button, Kitsu will grab all the frames with annotations and display the result. You can publish them with **Confirm**.
 
 ![Attachment snapshot detail](../img/getting-started/attachment_snapshot_detail.png)
 
+If you need to delete a line, click the pencil icon again. The cursor changes to a multi-directional cross ![Pencil](../img/getting-started/direction_arrow.png), and you can then select your line and move it around.
 
-If you need to delete your line, click the pencil icon again. The cursor changes to a multi-directional cross![Pencil](../img/getting-started/direction_arrow.png), and then you can select your line and move it around.
+<!-- Need to test this, couldn't get this to work -->
```

Comment:
I don't understand either what this refers too.

Reply:
Now I get it; line refers to a drawn element. Annotations are vectorized in Kitsu.

Reply:
Thanks, this was my mistake, managed to get it working now